.travis.yml

```diff
@@ -2,4 +2,5 @@ language: node_js
 node_js:
 - 10
```

Comment:
Should 10 just be dropped? or can we _just_ test 14?

Reply:
I'm not opposed to dropping 10 from our CI tests, but will let @shabnomnom weigh in as well.